src/MMLib.SwaggerForOcelot/Transformation/SwaggerJsonTransformer.cs

```diff
@@ -263,7 +263,7 @@ private static void TransformServerPaths(JObject openApi, string hostOverride)
             {
                 if (server[OpenApiProperties.Url] != null)
                 {
-                    server[OpenApiProperties.Url] = hostOverride.RemoveSlashFromEnd();
```

Comment:
Please, why you remove `.RemoveSlashFromEnd();`?

Reply:
Yes, you're right, I forgot to put at the end the line: .RemoveSlashFromEnd(); in the next pull request I'll include it

Reply:
You can make commit this change directly to your actual branch and push. This commit will be included in this PR.

Reply:
you can help me with the unit tests, I don't have so much experience in doing this kind of tests, I'm more used to test CRUDs with database than this kind of testing, I'm adding the documentation to the Readme.md file

Reply:
Ok. I'll write tests in next PR.